.github/workflows/delete-merged-branches.yml

```diff
@@ -0,0 +1,26 @@
+name: Branch Cleanup
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  delete-branch:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Delete merged branch
+        uses: SvanBoxel/delete-merged-branch@main
```

Comment:
Deleting looks like a simple gh cli action. Can we use something built-in instead of relying on others action?